core/templates/dev/head/components/ExplorationCreationButtonService.js

```diff
@@ -140,31 +140,38 @@ oppia.factory('ExplorationCreationButtonService', [
       showCreateExplorationModal: function(categoryList) {
         alertsService.clearWarnings();
 
-        siteAnalyticsService.registerOpenExplorationCreationModalEvent();
-
-        getModalInstance(categoryList, false).result.then(function(result) {
-          var category = $filter('normalizeWhitespace')(result.category);
-          if (!validatorsService.isValidEntityName(category, true)) {
-            return;
-          }
+        var currentPage = window.location.pathname;
```

Comment:
currentPathname might be a better name for this variable.

Though, actually, I'd suggest adding a getPathname() method to urlService in app.js, and using that instead. It's nice to keep all the URL stuff in one place.